assignments/prototypes.js

```diff
@@ -32,7 +53,48 @@
   * should inherit destroy() from GameObject through CharacterStats
   * should inherit takeDamage() from CharacterStats
 */
- 
+
+function Humanoid(attr) {
+  GameObject.call(this, attr),
+  CharacterStats.call(this, attr),
+  this.team = attr.team,
+  this.weapons = attr.weapons,
+  this.language = attr.language
+}
+
+Humanoid.prototype = Object.create(CharacterStats.prototype);
+Humanoid.prototype.greet = function() {
+  return `${this.name} offers a greeting in ${this.language}.`
+}
+
+function Hero(attr) {
+  Humanoid.call(this, attr),
+  this.attack = attr.attack
+}
+
+Hero.prototype = Object.create(Humanoid.prototype);
+Hero.prototype.death = function() {
+  if (this.healthPoints <= 0) {
+    return "you are dead";
+  } else {
```

Comment:
Based on what this method does (it seems to check if the `Hero` is dead or alive), you might want to rename it to reflect its behavior (some suggestions for renaming it: `.isAlive`, `.isDead`, `.checkLiving`, `.checkStatus`, etc.).